src/web/components/Content/Select/index.js

```diff
@@ -31,32 +33,33 @@ const initialData = {
 }
 
 class SelectContent extends Component {
-  state = initialData
+  state = initialState
 
   constructor(props) {
     super(props)
 
     window.botpress = window.botpress || {}
-    window.botpress.pickContent = (options = {}, callback) => {
-      this.setState({ step: formSteps.INITIAL }, () => {
-        this.searchContentItems()
-        this.props.fetchContentItemsCount()
-        this.props.fetchContentCategories()
-        this.callback = callback
-        this.setState({ show: true, activeItemIndex: 0 })
-        setImmediate(() => moveCursorToEnd(this.searchInput))
-      })
+    window.botpress.pickContent = ({ categoryId = null } = {}, callback) => {
```

Comment:
We should probably define this in a higher level component to allow multiple simultaneous pickers